Core/SaveState.cpp

```diff
@@ -433,12 +457,43 @@ namespace SaveState
 		}
 	}
 
+	bool UndoSaveSlot(const std::string &gameFilename, int slot) {
```

Comment:
Nothing calls this function, I guess it's here for future UI?

Reply:
Right, I just had trouble putting a button in a good place that didn't seem ugly or annoying.

-[Unknown]